api-gen/rendering/templates/class-member.tsx

```diff
@@ -0,0 +1,24 @@
+/** @jsx h */
```

Comment:
do we need these with the tsconfig options set? (`jsxFactory`)

Reply:
Apparently not, removed.